src/lib/API/api.js

```diff
@@ -211,8 +211,8 @@ class API {
    * `/send/magiclink` post api call
    * @param {string} magiclink
    */
-  sendMagicLinkByEmail(magiclink: string): Promise<$AxiosXHR<any>> {
-    return this.client.post('/send/magiclink', { magiclink })
+  sendMagicLinkByEmail(magiclink: string, fullName: string): Promise<$AxiosXHR<any>> {
+    return this.client.post('/send/magiclink', { magiclink, fullName })
```

Comment:
```suggestion
    return this.client.post('/send/magiclink', { magiclink })
```

Reply:
I pass the fullname, because we do not wait for the user/add, the fullname to appear in the private data only after the user/add is completed